include/nihstro/shader_bytecode.h

```diff
@@ -663,6 +663,13 @@ union SwizzlePattern {
         w = 3
     };
 
+    unsigned GetSelectorSrc(unsigned src) const {
```

Comment:
You could extract the bits yourself from the hex member and it would be (perhaps not noticeably, but still) faster